### DISCUSSION TITLE
Zookeeper dataDir and dataLogDir

### DISCUSSION THREAD

**Discussion by ganzah** [strimzi/strimzi-kafka-operator#6614] [Q&A] [answered]
Hi,

is there a way to  use different volumes for zookeeper dataDir and dataLogDir? I'm asking because I  believe that zookeeper dococumentation recommends having a dedicated device for it's transactional logs.

Thanks!

**Comment by scholzj** [comment, marked as answer]
No, I don't think we support it.

**Reply by ganzah** [reply]
ok, thanks for the reply. Out of curiosity, is there a recommendation on whether it's better to use a nfs volume or a iscsi volume for zookeeper?

**Reply by scholzj** [reply]
I'm not sure how NFS works with ZooKeeper TBH. I know it has issues with Kafka so I never used it for ZooKeeper either. So I personally would definitely go with iSCSI.

**Reply by ganzah** [reply]
ok, thanks!
